include/dt-bindings/pinctrl/esp-pinctrl-common.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_ESP_PINCTRL_COMMON_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_ESP_PINCTRL_COMMON_H_
+
+#define ESP32_PIN_NUM_SHIFT      0U
```

Comment:
nit: this header makes use of `BIT()`, so it should include `<sys/util_macro.h>`

Reply:
In this case, "BIT()" is one of those macros that is also defined on Espressif's HAL.

Reply:
I mean, header is not self-contained, this is a bad practice.